scratch-vm/src/extensions/noise/index.js

```
@@ -0,0 +1,200 @@
+// Perlin and simplex noise for all of us
+// MPL 2.0 License
+
+const iconURI =
```

ESLint

[error] 'iconURI' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)